src/containerapp/azext_containerapp/_params.py

```diff
@@ -81,6 +106,12 @@ def load_arguments(self, _):
         c.argument('user_assigned', nargs='+', help="Space-separated user identities to be assigned.")
         c.argument('system_assigned', help="Boolean indicating whether to assign system-assigned identity.")
 
+    with self.argument_context('containerapp create', arg_group='Container') as c:
+        c.argument('image', type=str, options_list=['--image', '-i'], help="Container image, e.g. publisher/image-name:tag.")
```

Comment:
> type=str

`str` is the default for `type`, so there is no need to declare it again

Reply:
@haroonf can you fix this.